packages/ui/src/forum/components/PostList/PostList.tsx

```diff
@@ -35,6 +35,9 @@ export const PostList = ({ threadId, isThreadActive, isLoading, isDiscussion }:
     (page: number) => history.replace({ pathname, search: page > 1 ? `page=${page}` : '' }),
     []
   )
+  useMemo(() => {
+    if(isReady && !posts.length) history.replace({pathname, search: ''});
+  },[isReady, posts])
```

Comment:
I don't understand. What is this change for ?
Also if this is needed please use a `useEffect` instead of `useMemo`.
Finally it looks like the linting wasn't applied.

Reply:
I think i tried useEffect at first but the whole page wasn't rendering properly. and the this is just to replace the history to the first  page if there's a case where someone enters a page number that probably doesn't have any posts

Reply:
@vrrayz 
I don't really understand this is a change on the PostList component which appears on forum threads, but this PR is about adding page numbers to the query params of the category page urls.

Also `posts.length` is never `0` because `posts` includes the thread initial post.

Unless I'm missing something, could you please remove this change. The rest of the PR is great.